Drivers/CacheControlDriver.cs

```diff
@@ -67,9 +67,9 @@ public override Task<IDisplayResult> DisplayAsync(ContentItem contentItem, Build
             return Task.FromResult<IDisplayResult>(null); ;
         }
 
-        private CacheControlPartSettings GetDefaultSettings(ContentItem contentItem)
+        private async Task<CacheControlPartSettings> GetDefaultSettings(ContentItem contentItem)

```

Comment:
As you've made this method async, you should suffix the method name with "Async", so it becomes `GetDefaultSettingsAsync`.